client/src/pages/Landing/Landing.js

```diff
@@ -53,20 +76,45 @@ const Landing = () => {
 
   return (
     <View style={styles.root}>
-      <StatusBar barStyle="light-content" />
-      <Text
-        fontWeight="regular"
-        color="blue.dark"
-        fontStyle="italic"
-        fontSize="6xl"
-      >
-        Landing
-      </Text>
-      <StyledButton
-        title="Login to app"
-        variant="primary"
-        onPress={loginUser}
-      />
+      <Image source={images.background_landing}></Image>
+      <View style={styles.button}>
+        <StyledButton
+          title="Continue with Google"
+          leftIcon={
+            <AntDesign name="google" size={24} color={colors.red.dark} />
+          }
+          variant="secondary"
+          onPress={loginUser}
+        />
+      </View>
+
+      <View style={styles.logo}>
+        <Logo height={160} width={160} />
+      </View>
+
+      <View style={styles.quote}>
+        <Text
+          fontWeight="regular"
+          color="white.dark"
+          style={{
+            fontFamily: 'GT_Haptik_bold',
+          }}
+          fontSize="3xl"
```

Comment:
Eventually, we will have more than one quote. To anticipate this change, you can save the selected quote to the local state of the Landing component with the `useState` hook.

So, you can do something like:

```
const [quote, setQuote] = useState("To speak a language...);
const [author, setAuthor] = useState("Frantz Fanon");
```